pkgs/package-overrides.nix

```diff
@@ -43,6 +43,35 @@ in
   };
 
   extensions = prev.extensions // {
+
+    mcrypt =
+      let
+        version = "0.5.6.40";
+        revision = "c37265eacdd0186cb3b0bfeb0e0104c8563807ef";
+      in
+        if lib.versionOlder prev.php.version "7.0" then
+          prev.mkExtension {
+            name = "mcrypt";
+            version = version;
+            src = pkgs.fetchFromGitHub {
+              owner = "php";
+              repo = "php-src";
+              rev = revision;
+              sha256 = "sha256-aDiGbOqy2L8Qqd6IliEN19Oa8JO1uDSEju6uUN4b0XI=";
+            } + "/ext/mcrypt";
+            nativeBuildInputs = [ pkgs.pkg-config ];
+            buildInputs = [ pkgs.libmcrypt ];
+            configureFlags = [ "--with-mcrypt=${pkgs.libmcrypt.outPath}" ];
+            doCheck = true;
+            checkTarget = "test";
```

Comment:
This is already hardcoded by `mkExtension`.

Reply:
I removed everything except `--with-mcrypt=` config flag. It doesn't work without that flag.

Reply:
`src` is still there though.

Reply:
Removed src